java/solr_plugins/RangedSearches/src/de/uni_tuebingen/ub/ixTheo/bibleDateRangeSearch/BibleDateRangeParserPlugin.java

```diff
@@ -0,0 +1,22 @@
+package de.uni_tuebingen.ub.ixTheo.bibleDateRangeSearch;
+
+
+import org.apache.solr.common.params.SolrParams;
+import org.apache.solr.common.util.NamedList;
+import org.apache.solr.request.SolrQueryRequest;
+import org.apache.solr.search.QParser;
+import org.apache.solr.search.QParserPlugin;
+
+
+public class BibleDateRangeParserPlugin extends QParserPlugin {
+
+    @Override
+    @SuppressWarnings("rawtypes")
+    public void init(final NamedList args) {
```

Comment:
at least call super here?

Reply:
We didn't need this in BibleRangeParserPlugin.java, so I'm not sure if this is needed/helpful here...  I mean then we wouldn't have needed to override the parent class' init in the first place.